content/services/services_powerdev.rst

```diff
@@ -20,36 +20,32 @@ via production and development (Fedora, OpenSUSE, and Debian) distributions.
 .. _Google: https://www.google.com
 .. _Chef: https://www.chef.io/chef/
 
-
 Members of the community can use these POWER servers to develop and test open
 source projects on the `Power Architecture`_ platform and in a `PowerLinux`_
 environment. These shared systems are intended for functional development and
-testing work, but are not ideal for performance testing.
+testing work, but are not ideal for performance testing. Developers looking for
+assistance can go to the `Linux on IBM Power Systems Developer`_ portal or `IBM
+Portal for OpenPOWER`_.
 
 .. _Power Architecture: http://en.wikipedia.org/wiki/Power_Architecture
 .. _PowerLinux: http://en.wikipedia.org/wiki/PowerLinux
+.. _Linux on IBM Power Systems Developer: https://developer.ibm.com/linuxonpower/
+.. _IBM Portal for OpenPOWER: https://www-355.ibm.com/systems/power/openpower/
 
-
-We offer either POWER7+ big endian instances using LPARs or POWER8 big or little
-endian instances running on KVM and providing access via OpenStack's API and GUI
-interface. The POWER8 instances offer much more flexibility, giving the ability
-to spin up or down instances on demand. We start projects out with a small
-quota, but can increase given resource availability and justification.
+We offer POWER8 big or little endian instances running on KVM and providing
+access via OpenStack's API and GUI interface. The POWER8 instances offer much
+more flexibility, giving the ability to spin up or down instances on demand. We
+start projects out with a small quota, but can increase given resource
+availability and justification.
 
 To request access to a POWER server, use our `request form`_ and we will get
 back to you shortly.
 
 .. _request form: /services/powerdev/request_hosting
 
-**Current Projects Hosted on POWER**
-
-.. csv-table:: Open Source Projects
-   :class: powerdev-tbl
-   :file: ./csv/powerdev_open_source_projects.csv
-   :widths: 20,80
+`Current Projects`_
 
-.. csv-table:: Academic Partners
-   :class: powerdev-tbl
-   :file: ./csv/powerdev_academic_partners.csv
-   :widths: 20,80
+`Former Projects`_
```

Comment:
I think it would make more sense to have a sentence explaining what these two links are about. In addition, I'd prefer to have these in a list so they have bullets.